### ISSUE TITLE
WebM file upload not working

### ISSUE THREAD

**Issue by k0nsl** [RocketChat/Rocket.Chat#1460]
I have tried every variant when it comes to whitelisted media types and uploading `WebM` doesn't work. At one point -- many weeks ago -- it worked. This was before the introduction of `Accepted Media Types` setting, I believe.
In my `File Upload` settings for `Accepted Media Types` I've currently got this: `image/*,audio/*,application/*,video/*`

When selecting a WebM file it brings up the dialogue but when you press "OK" to upload the file it gives the following message: `The file type is not accepted.`

Images, audio, PDFs and other files work fine. But not WebM.

**Comment by Sing-Li** [comment]
@k0nsl   Some server-side logs while you're attempting a webm upload will help.  TIA

**Comment by k0nsl** [comment]
@Sing-Li: that's just it, there's nothing in the terminal; no output whatever.

The only output given is in the channel itself (e.g `The file type is not accepted.`):
![test_tmp](https://i.knsl.org/2015-11-22_20-29-52.png)

**Comment by Sing-Li** [comment]
@k0nsl Awesome.  That'll help isolate our troubleshootin' to the client side [at first] ;-)

**Comment by k0nsl** [comment]
@Sing-Li: the dev-tools console doesn't give any clues; it remains the same after attempting to upload the .webm file.

**Comment by geekgonecrazy** [comment]
@k0nsl this just worked for me.  I added video/webm, I did have to restart the server when I increased the memory limit.. so its possible adding the file type requires restarting Rocket.Chat

**Comment by k0nsl** [comment]
@geekgonecrazy: I added the following into what is allowed to be uploaded (i.e almost anything is allowed): `image/*,audio/*,application/*,video/*,text/*`

I then proceeded to shutdown the meteor instance; pulled the newest code; started the application and tried uploading a WebM file. The result was 100% successful.

So, yes. A restart of the application is likely needed after changing or adding new file types.
